app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -72,18 +72,20 @@ def callback_for(provider)
 
       user_errors = @user.errors.full_messages
 
-      Rails.logger.error("Log in error: sign in failed. username: #{@user.username} - email: #{@user.email}")
-      Rails.logger.error("Log in error: auth data hash - #{request.env['omniauth.auth']}")
-      Rails.logger.error("Log in error: auth data hash - #{request.env['omniauth.error']&.inspect}")
-      Rails.logger.error("Log in error: user_errors: #{user_errors}")
+      Honeybadger.context({
+                            username: @user.username,
+                            user_id: @user.id,
+                            auth_data: request.env["omniauth.auth"],
+                            auth_error: request.env["omniauth.error"]&.inspect,
+                            user_errors: user_errors
+                          })
+      Honeybadger.notify("Omniauth log in error")
 
       flash[:alert] = user_errors
       redirect_to new_user_registration_url
     end
   rescue StandardError => e
-    Rails.logger.error("Log in error: #{e}")
-    Rails.logger.error("Log in error: auth data hash - #{request.env['omniauth.auth']}")
-    Rails.logger.error("Log in error: auth data hash - #{request.env['omniauth.error']&.inspect}")
+    Honeybadger.notify(e)
```

Comment:
The error will have all of the request and env info when we send it. I am not sure if it would above when we send a  string as an error so I added it to the context to be safe.